src/miral/regenerate-miral-symbols-map.py

```diff
@@ -411,7 +411,6 @@ def _report(publish, symbol):
     miral::toolkit::mir_input_event_get_pointer_event*;
     miral::toolkit::mir_input_event_get_touch_event*;
     miral::toolkit::mir_input_event_get_type*;
-    miral::toolkit::mir_input_event_has_cookie*;
```

Comment:
We should be bumping ABI, not dropping symbols without changing soname

Reply:
Yeah this might be one that I don't understand. Should I leave all of the `.map` files unchanged, or am I supposed to remove the symbols from them now?

Also, is there any simple mechanism for bumping the so names that need to be bumped?